backend/graphql/types/courseType.ts

```diff
@@ -8,6 +8,8 @@ const courseType = gql`
     image: String
     previewImage: String
     lessons: [ID!]!
+    private: Boolean
+    public: Boolean
```

Comment:
It would be good to call this property `published` as opposed to public.

Reply:
Done